src/firmware/app/primitives/autochip_move_primitive.c

```diff
@@ -0,0 +1,48 @@
+#include "firmware/app/primitives/autochip_move_primitive.h"
+
+#include <math.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+#include "firmware/app/primitives/move_manager.h"
+
+void app_autochip_move_primitive_start(AutochipMovePrimitiveMsg prim_msg,
+                                       void* void_state_ptr, FirmwareWorld_t* world)
+{
+    app_move_manager_start(void_state_ptr, world, prim_msg.position_params,
+                           prim_msg.final_angle.radians);
+
+    const FirmwareRobot_t* robot = app_firmware_world_getRobot(world);
+
+    Dribbler_t* dribbler = app_firmware_robot_getDribbler(robot);
+    Chicker_t* chicker   = app_firmware_robot_getChicker(robot);
+    app_dribbler_setSpeed(dribbler, (uint32_t)prim_msg.dribbler_speed_rpm);
```

Comment:
Should we spin the dribbler in the `autochip` primitive? Seems like we could damage the roller more by dribbling while chipping.

Reply:
hm, how are we supposed to do backspin chipping then?

Reply:
@MathewMacDougall for his input. Not a blocking comment, just a thought. You're right that we would need this for backspin

Reply:
1. This code is not backspin chipping (as I think of it). There's no time for the ball to get up to speed so we won't build up any backspin so there'd be no noticeable effect on the ball when it lands

2. Unsure about how much damage this may cause. Using the dribbler is a reasonable idea to try get more _consistent_ chips because hopefully it will put the ball in a more consistent position so it makes better contact with the chipper (inconsistency typically comes from the ball being too far away or too close). This may require testing in different scenarios to see if it's worth it. In most cases where we'd use this there should be relative velocity between the ball and robot on contact (robot moves into ball, or ball moves into robot). This _should_ result in better contact without the need for the dribbler so perhaps it's not needed. Also depending on how the dribbler behaves and when the breakbeam fires, a running dribbler may knock the ball away if it doesn't catch it properly for some reason.

tl;dr we need to do tests to check for dribbler damage, and consistency gains / losses.